app/component/map/popups/RoadworksPopup.js

```diff
@@ -0,0 +1,71 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Relay from 'react-relay/classic';
+import moment from 'moment';
+import Card from '../../Card';
+import CardHeader from '../../CardHeader';
+import { station as exampleStation } from '../../ExampleData';
```

Comment:
What is that needed for?

Reply:
A leftover from a copy&past josb. I removed it.